extensions/blocks/publicize/twitter/options.js

```diff
@@ -74,16 +76,26 @@ const PublicizeTwitterOptions = ( {
 				{ __( 'Twitter settings', 'jetpack' ) }
 			</h3>
 			<RadioControl
+				className="jetpack-publicize-twitter-options__type"
 				selected={ isTweetStorm ? 'tweetstorm' : 'single' }
 				options={ [
 					{
-						label: __( 'Share your blog post as a link in a single tweet', 'jetpack' ),
+						label: (
```

Comment:
I'm not super excited about this workaround, but `RadioControl` doesn't expose a prop to do what we want, and `PostVisibility` in Gutenberg works around this by re-implementing the entire thing.